File: Upcoder/Other/LTCB_GK_cau3/LTCB_GK_cau3.cs
using System;
using System.Collections.Generic;
namespace LTCB
{
    public class Program
    {
        static bool IsPrime(int n)
        {
            if (n < 2)
                return false;
            for (int i = 2; i * i <= n; i++)
                if (n % i == 0)
                    return false;
            return true;
        }
    
        static void Main(string[]args)
        {
            int n = int.Parse(Console.ReadLine());
    
            if (n < 2)
                Console.WriteLine(-1);
            else
            {
                List<string> primes = new List<string>();
                for (int i = 1; i <= n; i++)
                {
                    if (IsPrime(i))
                    {
                        primes.Add(i.ToString());
                    }
                }
                Console.Write(string.Join(" ", primes));
            }
        }
    }   
}
    

File: Upcoder/Other/LTCB_GK_cau3/LTCB_GK_cau3.py
def is_prime(n):
    if n < 2:
        return False
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0:
            return False
    return True

n = int(input())
if n < 2:
    print(-1)
else:
    primes = []
    for i in range(1, n + 1):
        if is_prime(i):
            primes.append(str(i))
    print(" ".join(primes), end="")
